### DISCUSSION TITLE
SSL connection between app and a mysql server

### DISCUSSION THREAD

**Discussion by darul75** [prisma/prisma#15316] [Q&A] [answered]
Hi all,

I would need to enable an SSL connectivity, i.e. our DB expecting to use an SSL connection to it. 

In this case, we would need to provide Prisma with the database’s CA so that we know we are connecting to the actual DB and not something intercepted.

I have tried to put a CA certificate .crt file into `/etc/ssl` and then use this extra parameters

`'&sslmode=require&sslcert=../../../../../../etc/ssl/ca.crt'`

using pnpm Prisma being into

`/app/node_modules/.pnpm/@prisma+client@3.11.1_prisma@3.11.1/node_modules/@prisma`

it is unclear from where the relative path should be set...

and if I got it right also.

**Comment by andrewicarlson** [comment, marked as answer]
@darul75 Root certificates are resolved relative to the `prisma` folder (for example, where your `schema.prisma` lives). You can find more about [securing database connections](https://www.prisma.io/docs/concepts/components/prisma-schema/data-sources#securing-database-connections) and the [MySQL SSL connection specifically](https://www.prisma.io/docs/concepts/database-connectors/mysql#configuring-an-ssl-connection)

**Reply by darul75** [reply]
Thank you @andrewicarlson had read this documentation and will read it again, for now we adopted the less `sslaccept` restricted choice. We will follow up later with that once other things are sorted.

**Reply by andrewicarlson** [reply]
@darul75 cool – if you run into other problems or would like to chat more, feel free to join us on [Slack](slack.prisma.io/) – my username is the same there as it is here on GitHub.
